test/compaction-helpers.test.ts

```
@@ -0,0 +1,45 @@
+import { applyCompactionIfNeeded } from "../lib/request/compaction-helpers.js";
+import { applyCompactionIfNeeded } from "../lib/request/compaction-helpers.js";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'applyCompactionIfNeeded'. Consider to delete it or rename it.

'applyCompactionIfNeeded' is defined here:


(lint/suspicious/noRedeclare)